### ISSUE TITLE
`SortDirection` field of ImGuiTableColumnSortSpecs doesn't appear in the wrapper

### ISSUE THREAD

**Issue by pthom** [pthom/imgui_bundle#42]

**Comment by dcnieho** [comment]
Yes, its a different issue from the other one, different struct.

**Comment by pthom** [comment]
`SortDirection`is a bitfield member!
 
````cpp
// Sorting specification for one column of a table (sizeof == 12 bytes)
struct ImGuiTableColumnSortSpecs
{
    ImGuiID                     ColumnUserID;       // User id of the column (if specified by a TableSetupColumn() call)
    ImS16                       ColumnIndex;        // Index of the column
    ImS16                       SortOrder;          // Index within parent ImGuiTableSortSpecs (always stored in order starting from 0, tables sorted on a single criteria will always have a 0 here)
    ImGuiSortDirection          SortDirection : 8;  // ImGuiSortDirection_Ascending or ImGuiSortDirection_Descending (you can use this or SortSign, whichever is more convenient for your sort function)

    ImGuiTableColumnSortSpecs() { memset(this, 0, sizeof(*this)); }
};
````

As such, it cannot be shared as a property!

**Comment by pthom** [comment]
Fixed with 98f1c6b6

**Comment by dcnieho** [comment]
Thanks, works!
